config/loader/aws/snowflake.config.reference.hocon

```diff
@@ -62,6 +62,16 @@
     #  # List of necessary permissions needs to be given to role specified in the following link:
     #  # https://docs.snowflake.com/en/user-guide/data-load-s3-config-aws-iam-user.html
     #  "roleArn": "arn:aws:iam::123456789:role/role_name"
+    #   
+    #  # If 'TempCreds' load auth method is used, those values will be used as a session duration
+    #  # of temporary credentials. In that case, it can't be greater than maximum session duration
+    #  # of IAM role used for temporary credentials and can't be less than 15 minutes.
+    #  credentialsTtl {
+    #   # Session duration for loading (actual COPY statements) queries.
+    #   "loading": "2 hours"
+    #   # Session duration for non-loading (such as ALTER TABLE or metadata queries) queries.
+    #   "nonLoading": "15 minutes"
```

Comment:
This is not quite right -- the other session is used for the folder monitoring feature only.  There's a step of the feature where we need to copy a file from s3 into Redshift, and for that we need some credentials.  We don't generate temporary credentials for things like `ALTER TABLE`.

For this reason, I would prefer it if `credentialsTtl.loading` and `credentialsTtl.nonLoading` could be renamed to something like `credentialsTtl.loadingEvents` and `credentialsTtl.folderMonitoring`.

For that matter.... maybe it's making the configuration file over-complicated to have two different ttl configurations.  What do you think of having a single `credentialsTtl` parameter, and using the same value for both session types?

Reply:
Yes! I like the idea of a single `credentialsTtl` parameter, I'll do that.